src/components/Confirm/index.js

```diff
@@ -234,6 +234,6 @@ const FaceConfirm = (props) =>
   <MapConfirm {...props} method= 'face' />
 
 const DocumentFrontConfirm = appendToTracking(DocumentFrontWrapper, 'front')
-const DocumentBackConfrim = appendToTracking(DocumentBackWrapper, 'back')
+const DocumentBackConfirm = appendToTracking(DocumentBackWrapper, 'back')
```

Comment:
The old string and the new string look identical to me... 🤔

Reply:
I know they aren't identical cause github can detect the change. I was just wondering what the difference is because I can't see it 😄

Reply:
DocumentBackCon*fri*m vs DocumentBackCon*fir*m 😄

Reply:
Ahah, I can see it now!